AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -555,6 +555,26 @@ - (void)reloadRowsAtIndexPaths:(NSArray *)indexPaths withAnimationOptions:(ASDat
   }];
 }
 
+- (void)relayoutAllRows
+{
+  [self performEditCommandWithBlock:^{
+    ASDisplayNodeAssertMainThread();
+    LOG(@"Edit Command - relayoutRows");
+    [_editingTransactionQueue waitUntilAllOperationsAreFinished];
+    
+    NSArray *indexPaths = ASIndexPathsForMultidimensionalArray(_completedNodes);
+    NSArray *loadedNodes = ASFindElementsInMultidimensionalArrayAtIndexPaths(_completedNodes, indexPaths);
```

Comment:
These are quite expensive calls.  Can we avoid them and use iteration over _completedNodes directly?  See other methods in ASDataController that use NSMutableIndexPath, or NSIndexSet